tests/test_device_types.py

```diff
@@ -101,5 +101,5 @@ def test_equality(self):
     def test_string_representations(self):
         """Test that the string representations are unique."""
         all_device_types = xcrun.simctl.listall.device_types()
-        strings = set([str(device_type) for device_type in all_device_types])
+        strings = set([device_type.__str__() for device_type in all_device_types])
```

Comment:
(Just playing with suggestions :P ) 

```suggestion
        strings = set([str(device_type) for device_type in all_device_types])
```